code/modules/tgui/tgui_input_text.dm

```diff
@@ -50,10 +52,12 @@
  * * default - The default (or current) value, shown as a placeholder.
  * * max_length - Specifies a max length for input.
  * * multiline -  Bool that determines if the input box is much larger. Good for large messages, laws, etc.
+ * * encode - If toggled, input is filtered via html_encode. Setting this to FALSE gives raw input.
+ * * no_trim - Like stripped_input's no_trim. Boolean determines whether to use copytext or trim
```

Comment:
https://github.com/tgstation/tgstation/blob/master/.github/guides/STYLE.md#avoid-negative-variable-names

Change to something that reads positively, like `use_copytext`, or better yet, change it to something like `bikeshed = BIKESHED_COPYTEXT` or `bikeshed = BIKESHED_TRIM`.

Also explain why you might want this.